docs/articles/apex/writing.md

```diff
@@ -2,18 +2,29 @@
 title: Write Apex Code
 ---
 
-View outlines of Apex classes and triggers, see code-completion suggestions, and find syntactic errors in your code. The Apex extension is powered by the [Apex Language Server](language-server).
+Use the Apex extension for VS Code to access code-editing features such as code completion, go to definition, view outlines of Apex classes and triggers, refactoring, and find syntactic errors in your code.
 
-## View Code-Completion Suggestions
+## Code-Completion
 
-To see code-completion suggestions, press Ctrl+space when you’re working in a `.cls` or `.trigger` file. To navigate between the suggestions, use the arrow keys. To auto-complete a suggestion from the list, press Enter. To change how suggestions are pre-selected, see [IntelliSense - Suggestion selection](https://code.visualstudio.com/docs/editor/intellisense).
-![Animation showing code completion of a System.debug() statement](../../images/apex_completion.gif)
+Apex extension provides context-sensitive suggestions when you’re working in an Apex class or trigger. As you type, the auto-completion list of members such as, methods or variables pop up. The auto-completion list also displays the documentation for the suggestions. Use these keys:
 
-## Insert Code Snippets
+- Ctrl+space to view see code-completion suggestions
+- Arrow keys to navigate between the suggestions
+- Enter key to select from the suggestions
 
-To see available Apex code snippets when you’re working in a `.cls` or `.trigger` file, run **Insert Snippet**. Snippets are available for class and interface definitions, a variety of statements, and much more. These code snippets are also available as code-completion suggestions.
+![Animation showing code completion of PropertyController](../../images/apex_completion_with_doc.gif)
 
-## View or Jump to Definitions
+If you want to change how suggestions are pre-selected, see [IntelliSense - Suggestion selection](https://code.visualstudio.com/docs/editor/intellisense).
```

Comment:
```suggestion
If you want to change how suggestions are pre-selected, see [IntelliSense - Suggestion selection](https://code.visualstudio.com/docs/editor/intellisense#_customizing-intellisense).
```

Minor change but this will take folks directly to the section on how to customize intellisense.